src/seaport.ts

```diff
@@ -2273,8 +2199,9 @@ export class OpenSeaPort {
       salt: WyvernProtocol.generatePseudoRandomSalt(),
       metadata: {
         bundle,
-        schema: schema.name as WyvernSchemaName,
-      },
+        // TODO deprecate
+        schema: schemas[0].name as WyvernSchemaName
```

Comment:
should we null check here?

Reply:
no need, it's deprecated and checked earlier (there's a check that you can't have fewer than one asset/schema in getWyvernAsset)